src/messages/manifestjson.js

```diff
@@ -166,6 +166,15 @@ export function manifestContentScriptFileMissing(path, type) {
   };
 }
 
+// https://github.com/mozilla/addons-linter/issues/1650
+// Potentially temporary
+export const MANIFEST_CSCRIPT_INVALID = {
+  code: 'MANIFEST_CSCRIPT_INVALID',
+  message: _('Invalid "matches" entry in "content_script".'),
```

Comment:
Let's go with:

message: Forbidden content found in add-on.
description: This add-on contains forbidden content.